packages/text-annotator/src/model/TextAnnotation.ts

```diff
@@ -12,10 +12,26 @@ export interface TextAnnotationTarget extends AnnotationTarget {
 
 }
 
-export interface TextSelector {
+/**
+ * Matches the `Text Quote Selector` spec
+ * @see https://www.w3.org/TR/annotation-model/#text-quote-selector
+ */
+export interface TextQuoteSelector {
```

Comment:
A bit like my comment concerning the LifecycleInfo type: I don't think there's a benefit to split the TextSelector into two parts. They won't ever be handles in separate in the core model. Whether WebAnno treats them differently is another matter, and IMO not necessarily a concern for the core model.